lang/ilias_en.lang

```diff
@@ -4555,17 +4555,26 @@ common#:#mail_at_the_ilias_installation#:#You received %1$s new mail at the ILIA
 common#:#mail_attachment#:#Mail attachment
 common#:#mail_b_inbox#:#Inbox
 common#:#mail_c_trash#:#Trash
+common#:#mail_cron_scheduled_mails#:#Send scheduled mails.
+common#:#mail_cron_scheduled_mails_desc#:#Sends emails from users according to individually defined times.
```

Comment:
I suggest:

- Sends scheduled user mails according to their defined delivery times.
- Sends user mails at their individually defined times.